test/Microsoft.Extensions.Logging.Test/LoggerTest.cs

```diff
@@ -26,7 +26,7 @@ public void Log_IgnoresExceptionInIntermediateLoggersAndThrowsAggregateException
             // Assert
             Assert.Equal(new[] { "provider1.Test-Hello!", "provider3.Test-Hello!" }, store);
             Assert.NotNull(aggregateException);
-            Assert.Equal("An error occurred while writing to logger(s).", aggregateException.Message);
```

Comment:
These are in preparation for newer CoreCLR packages that changed the Message overload of AggregateException https://github.com/dotnet/coreclr/commit/15db7abcb1aaec8f29f0029bc73632494dc281b3